Duplicati/Library/Main/Operation/CompactHandler.cs

```diff
@@ -98,11 +98,9 @@ internal bool DoCompact(LocalDeleteDatabase db, bool hasVerifiedBackend, ref Sys
 				using(var bk = sharedBackend == null ? new BackendManager(m_backendurl, m_options, m_result.BackendWriter, db) : null)
                 {
                     var backend = bk ?? sharedBackend;
-                    if (!hasVerifiedBackend && !m_options.NoBackendverification)
+                    if (!hasVerifiedBackend)

```

Comment:
Does this change the logic? Or does the calling code now set the flag according to the user options?

Reply:
I moved the check for `!options.NoBackendverification` into the new `FileListProcessor.VerifyRemoteList` overload.